modelhub/modelhub/pipelines/util.py

```diff
@@ -18,6 +18,7 @@ def get_objectiv_data(
     with_sessionized_data: bool = True,
     identity_resolution: Optional[str] = None,
     anonymize_unidentified_users: bool = True,
+    global_contexts: List[str] = []
```

Comment:
:P let's avoid mutable default arguments
```suggestion
    global_contexts: List[str] = None
```

Reply:
oh noes. :(